### ISSUE TITLE
`--config .sass-lint.yml` gives `undefined:1 files:`

### ISSUE THREAD

**Issue by stephanebruckert** [srowhani/sass-lint-auto-fix#3]
## Problem Description

`--config .sass-lint.yml` gives `undefined:1 files:`

## Steps to recreate

- put `.sass-lint.yml` at the root of the project with following content:

```
files:
  include: '**/*.s+(a|c)ss'
syntax:
    include:
      - scss
      - sass
resolvers:
  property-sort-order: 1
  attribute-quotes: 1
  border-zero: 1
  no-color-keywords: 1
  no-css-comments: 0
  no-important: 0
  no-trailing-zero: 1
  space-after-bang: 1
  space-before-bang: 1
  space-after-colon: 1
  space-before-colon: 1
  hex-length: 1
```

 - run `sass-lint -c .sass-lint.yml` and see that it works
 - run `sass-lint-auto-fix -c .sass-lint.yml` and get an error:

```
undefined:1
syntax:
^

SyntaxError: Unexpected token s in JSON at position 0
    at JSON.parse (<anonymous>)
    at /usr/local/lib/node_modules/sass-lint-auto-fix/dist/index.js:162:44
    at /usr/local/lib/node_modules/sass-lint-auto-fix/dist/index.js:173:7
    at /usr/local/lib/node_modules/sass-lint-auto-fix/dist/index.js:3:68
    at Object.<anonymous> (/usr/local/lib/node_modules/sass-lint-auto-fix/dist/index.js:6:2)
    at Module._compile (module.js:649:30)
    at Object.Module._extensions..js (module.js:660:10)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:501:12)
    at Function.Module._load (module.js:493:3)
```

Hope you can help, and thank you so much for your work on this repo!

**Comment by srowhani** [comment]
Thank you for reporting! I'll address this soon

**Comment by srowhani** [comment]
Should be resolved by f4b1bfbc8e8334fa297ccb8b1e5db49e1299439a. Published a new version as `0.2.3`.

**Comment by stephanebruckert** [comment]
Nice, thanks it's all good now
